lib/asm-parser.js

```diff
@@ -224,7 +226,8 @@ class AsmParser extends AsmRegex {
         asmLines.forEach(line => {
             let match;
             if (line.trim() === "") {
-                result.push({text: "", source: null});
+                if (!this.removeBlankLines)
```

Comment:
Can you explicitly initialise `removeBlankLines` in the constructor please? I prefer not to rely on not-there/`undefined` values in JS where possible.

Reply:
Done.